data/bookkeeping/block.go

```diff
@@ -254,6 +254,20 @@ func (block *Block) Seed() committee.Seed {
 	return block.BlockHeader.Seed
 }
 
+// Compressed returns the Block without ApplyData
+func (block *Block) Compressed() Block {
+	c := *block
+	c.Payset = nil
+	for _, stb := range block.Payset {
```

Comment:
do we really need to copy payset?

Reply:
We want to create two "versions" - one for sending on the network, and one that would be used locally if this proposal is accepted by the network.
The copy is "stripping" the Payset from the ApplyData.

Reply:
But you're correct that we should make sure we're doing that exactly one time.